pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -361,21 +366,6 @@ func (vm *windows) waitForHNSNetworks() error {
 	return errors.Wrap(err, "timeout waiting for OVN overlay HNS networks")
 }
 
-// waitForHybridOverlayToRun waits for the hybrid-overlay-node.exe to run until the timeout is reached
```

Comment:
Don't we need to wait for the service to go to running or do you think it waiting for HNS networks covers that? It will help us debug if we can be more specific regarding failures, i.e. the service did not go to running vs HNS networks did not get created.

Or does the service not go to running if there is an error in creating the HNS networks?

Reply:
I did think that waiting for HNS networks covers the scenario and we do not need to wait or retry more although I agree it would be easier to debug if we have that, AFAIK the code in hybrid-overlay-node the service will go to running state even with HNS network errors so we definitely need to waitForHNSNetworks

Reply:
I have added this method back with changes required for checking is the service is running.

Reply:
Could we make this function more generic/re-usable by passing the service name as param? Something like:
```
func (vm *windows) waitForServiceToRun(serviceName string) error {
```
wdyt?

Reply:
@LorbusChris that seems reasonable considering we could reuse this function for other services in the future. Thanks for the suggestion!